src/app/components/books/books.scss

```diff
@@ -0,0 +1,49 @@
+$yellow-star: #EAC435; 
+
+.glyphicon {
```

Comment:
One of the guys from ui.bootstrap recommended changing the template to use font awesome instead of glyphicon, which I did and it worked. I removed the css hack.